lib/govuk_navigation_helpers/related_items.rb

```diff
@@ -32,6 +33,25 @@ def related_items
 
     attr_reader :content_item
 
+    def register_to_vote_section
+      return unless content_item.document_type == 'completed_transaction'
+      return unless promoting_register_to_vote?
+
+      {
+        title: 'Register to vote',
+        description: 'To vote in the General Election on 8 June, you need to apply to register by 11:59pm on 22 May.',
```

Comment:
I don't know anything about promotions - will the promotion be automatically removed at a certain time?